providers/sample/src/java/org/sakaiproject/provider/user/SampleUserDirectoryProvider.java

```diff
@@ -305,7 +308,7 @@ public Collection findUsersByEmail(String email, UserFactory factory)
 		UserEdit edit = factory.newUser();
 
 		// assume a "@local.host"
```

Comment:
This comment should be changed/removed.
Anyway this should be merged even with this comment :smile: